helpers/cdn.js

```diff
@@ -10,11 +10,12 @@ const factory = globals => {
         let fullPath = cdn(path);
 
         const options = arguments[arguments.length - 1];
-        if (options.hash.resourceHint) {
+        if (options.hash && options.hash.resourceHint) {
```

Comment:
🍹  consider https://bobbyhadz.com/blog/javascript-check-if-object-has-nested-property

(not blocking, just nice)